### ISSUE TITLE
What's the workaround for "ChatGPT is at capacity" feature?

### ISSUE THREAD

**Issue by LJM2018** [nezort11/chatgpt-mobile#1]
First of all, thanks for creating an app that helps using ChatGPT. 

One of the features listed on the Readme says "chatgpt is at capacity" workaround", however, I didn't see how I could access it. 

Even if I am a plus subscriber, when I get a login link and click, it directs me to the browser instead of the app. In other words, I can't use ChatGPT with this app when OpenAI is at capacity.

**Comment by nezort11** [comment]
hi, thanks. i found that visiting https://chat.openai.com/auth/ext_callback?next= (instead of https://chat.openai.com) often redirects to login page (even if not it is at capacity). I also use VPN, reload page many times + clear storage (it helps sometimes) . is is basically it

it will be probably be fixed in the future (if not already, i haven't seen chatgpt is at capacity for long time to test)

**Comment by LJM2018** [comment]
No, I tried the latest release, however, the "at capacity" page still takes over the entire screen and there is no refresh button. Not sure if it is a phone problem(I am using a Huawei mate 40 pro). 

And it appears that a new issue comes up, it logs me out if I close the app). But in release 1.0.1, it keeps my login status even if I kill the app.
